tests/addMany-strip-extensions-mock/plopfile.js

```diff
@@ -21,13 +21,15 @@ module.exports = function (plop) {
 				destination: 'src/',
 				stripExtensions: ['hbs'],
 				templateFiles: 'plop-templates/remove-hbs/*',
-				abortOnFail: true
+				abortOnFail: true,
+				globOptions: { dot: true },
 			},{
 				type: 'addMany',
 				destination: 'src/',
 				stripExtensions: true,
 				templateFiles: 'plop-templates/remove-all/*',
-				abortOnFail: true
+				abortOnFail: true,
+				globOptions: { dot: true },
```

Comment:
Is this a required change in old code projects?

If so, we may want to include it in a major file bump that we're likely going to release soon (not huge changes, just minor breaking ones)

Reply:
I'm not sure what you mean by 'old code projects' sorry, but this is required as without it plop won't copy across the dot file, and so we can't test if it's hbs extension has been stripped. It shouldn't affect any pre-existing part of the test, but you might like to wait until the next major file bump to be on the safe side.

Reply:
I think this change requires a major version bump, because the result is different from what is was previously, so no bugfix or added feature. That being said, I need this function as well, so it would be nice to have it included asap.